gensim/test/test_fasttext.py

```diff
@@ -42,6 +41,17 @@ def __iter__(self):
                 yield utils.simple_preprocess(line)
 
 
+#
+# Not sure why these tests are hanging in my development environment.
+# They seem to be passing on CI.
+#
+SKIP_CORPUS_FILE = (
+    os.environ.get('DISABLE_CORPUS_FILE_TRAINING'),
```

Comment:
default value if `DISABLE_CORPUS_FILE_TRAINING` not set?

Reply:
I'll get rid of this, we don't need it anymore.